src/celeritas/em/interactor/MuBetheBlochInteractor.hh

```diff
@@ -0,0 +1,257 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/em/interactor/MuBetheBlochInteractor.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <cmath>
```

Comment:
This may be a silly question, but do we have a reason to use `cmath` vs. `math.h`?